packages/application-shell/src/components/navbar/use-navbar-state-manager.ts

```diff
@@ -135,7 +136,8 @@ const useNavbarStateManager = (props: HookProps) => {
   const checkSize = useCallback(
     throttle(() => {
       const shouldOpen = window.innerWidth > 1024;
-      const canExpandMenu = window.innerWidth > 918;
+      const canExpandMenu =
+        window.innerWidth > (props.isNewNavigationEnabled ? 1200 : 918);
```

Comment:
I was wondering if we should move these values to the package [constants file](https://github.com/commercetools/merchant-center-application-kit/blob/main/packages/application-shell/src/constants.ts#L1).